apps/docs/src/guide/predicates/methods.md

```diff
@@ -10,15 +10,15 @@ The `Predicate` class extends the [`Account`](https://docs.fuel.network/docs/fue
 
 ### `getBalances`
 
-This will return the balances of the wallet that is being used to interact with the predicate.
+This will return the balances of the account that owns the resources in the predicate.
```

Comment:
```suggestion
This will return the balances of all assets owned by the predicate.
```
The predicate itself owns the resources and _it_ is the account.